src/service-worker.js

```diff
@@ -0,0 +1,29 @@
+var cacheName = 'invoizr'
```

Comment:
I did it with the first one plugin, take a look. If it doesn't match I will change it.